src/SDKs/Azure.Base/data-plane/Azure.Base/Http/HttpMethodConverter.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Base.Http
+{
+    public static class HttpMethodConverter
```

Comment:
You may want to consider naming this `HttpPipelineMethodConverter` as it's not acting on `System.Net.Http.HttpMethod` and may be confusing.